action.yaml

```diff
@@ -32,3 +35,14 @@ runs:
         LAVA_VERSION: ${{ inputs.version }}
         LAVA_CONFIG: ${{ inputs.config }}
         LAVA_FORCECOLOR: ${{ inputs.forcecolor }}
+    - name: Add summaries
+      uses: actions/github-script@v7
+      if: ${{ !cancelled() }}
+      env:
+        RESULT: ${{ steps.lava.outputs.report }}
+        METRICS: ${{ steps.lava.outputs.metrics }}
+        COMMENT_PR: ${{ inputs.comment-pr }}
+      with:
+        script: |
+          const utils = require(`${process.env.GITHUB_ACTION_PATH}/utils.js`);
+          await utils.process({github, context, core});
```

Comment:
`utils.process` seems too generic. Neither the module name nor the function name provide any useful information about what it does. What about renaming the module to `report` and exporting the functions `writeSummary` and `postComment`?

Reply:
done